analyzer/src/main/kotlin/managers/SBT.kt

```diff
@@ -42,6 +41,7 @@ class SBT(config: AnalyzerConfiguration) : PackageManager(config) {
             listOf("build.sbt", "build.scala")
     ) {
         private val VERSION_REGEX = Regex("\\[info]\\s+(\\d+\\.\\d+\\.[^\\s]+)")
+        private val PROJECT_REGEX = Regex("\\[info] \t [ *] (.+)")
```

Comment:
Since we rely on having the `info`-level log messages, it might be beneficial to set the desired log level when we launch sbt, as described here: https://www.scala-sbt.org/1.0/docs/Howto-Logging.html, or, concretely, via `sbt --info`.

Reply:
Good point, I'll add that as a new commit to this PR.